features/configuring_gateway_network_topology.md

```diff
@@ -121,9 +121,11 @@ https://istio.io/latest/docs/ops/configuration/traffic-management/network-topolo
 
 **Approvals**
 
-- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
-- [ ] The TOC has reviewed and approved promotion of the feature as part of the
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
```

Comment:
@jacob-delgado is this ready for alpha? It looks like everything is checked at this point. If so, can you also update [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml)?

Reply:
Updated features.yaml